app/main.ipynb

```diff
@@ -126,19 +166,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "\n",
+    "subseq = array[[0, 1, 4, -1]]\n",
+    "subseq.sum()"
```

Comment:
The subsequence selection and summation are correct, resulting in the expected sum of 27. Ensure that the indices used match the task requirements.